com.unity.cinemachine/Editor/Upgrader/CinemachineUpgradeManager.cs

```diff
@@ -426,6 +421,49 @@ static void SynchronizeComponents(GameObject prefabInstance, GameObject converte
                 }
             }
         }
+        
+        void UpgradeAnimationTrackReferences()
+        {
+            for (var s = 0; s < m_SceneManager.SceneCount; ++s)
```

Comment:
We're opening all the scenes over and over again.  I bet this is really slow.  I wonder if this module can be re-organized to iterate through the scenes way fewer times.  This is not high-priority, but it's worth thinking about for a future commit.